about.html

```diff
@@ -0,0 +1,174 @@
+            <a href="facebook.com/paulmalongo"><i class="fa-brands fa-facebook-f"></i></a>
+            <a href="facebook.com/paulmalongo"><i class="fa-brands fa-twitter"></i></a>
+            <a href="#">English</a>
+            <a href="https://canon20paul.github.io/My-Portfolio/">My Page</a>
+            <a href="#">Logout</a>
+        </nav>
+        <nav class="nav-bar">      
+          <div class="nav-item logo">
+               <a href="#"><img class="logo-img" src="images/proLogo.png" alt="logo"/> 
+            <h3>REMOTE WORKERS  SUMMIT<br>2022</h3></a>
+          </div>
+           <ul class="list-menu">
+            <li class="nav-item">
+                <a href="index.html" class="nav-link">Home</a>
+            </li>
+            <li class="nav-item">
+                <a href="about.html" class="nav-link">About</a>
+            </li>
+            <li class="nav-item">
+                <a href="" class="nav-link">Program</a>
+            </li>
+            <li class="nav-item">
+                <a href="" class="nav-link">Join</a>
+            </li>
+               <li class="nav-item labelT">
+              <a href="" class="nav-link">WFH Remoters..</a>
+          </li><img src="images/crane.jfif" alt="" class="crane1"> <img src="images/crane1.jfif" alt="" class="crane2">
+          </ul>
+        <div class="hamburger">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+        </div>
+        </nav>
+        </header>
+        <!--End of Header section-->
+    </header>
+    <!--Header section-->
+    <section class="mainS"></section>
+           <div class="main-header">
+            <p class="hello">"Hello! Am a WFH"</p>
+                   <h1 class="header-text">
+                        REMOTE DEV'S ANNUAL GLOBAL &nbsp;SUMMIT 2022 
+          </h1>
+          <p class="desc"> The after month of covid 19 saw an over whelming growth of the demand for remote workers
+            Previously a very slow sector experienced a sharp demand as companies had also come to appreciate thie benefits associtaed. the initially 
+            unknown event now gathers all works of life with a turn up of thousands each year. For the first Time, its in AFRICA , Kampala, Uganda in E.Africa this OCTOBER</p>
+             <p class="desc2">Please contact @www.MunyonyoCommonwealth esort.ug, Along Lake Victoria Shores, @www.TripeAdvisors.com, www.Eaafricansafari.com</p>
+             </div>
+        <div class="middle">
+             <div class="mid-label">
+              <!-- <div class="footer-head ">
+                <a href="#"> 
+                  <img class="logo-img" src="images/proLogo.png" alt="logo"/></a>
+                   <p>
+                Ⓒ 2022 WFH Uganda <br />
+                All Rights Reserved
+              </p>
+              </div> -->
+            
+                  <!-- <h3>REMOTE <br>GLOBAL <br>SUMMIT</h3> -->
+                  
```

Comment:
- [ ] Kindly remove this commented code to improve readability and make the project more professional 💯